src/types/docs-schemas/resources.schema.json

```diff
@@ -58,6 +58,22 @@
               "$ref": "definitions.schema.json/#/definitions/sentinel",
               "format": "uri-reference"
             }
+          },
+          "deployment-configs": {
+            "title": "Deployment Configs",
+            "type": "object",
+            "additionalProperties": {
+              "$ref": "definitions.schema.json/#/definitions/deployment-config",
```

Comment:
Does this not need to be deploymentConfig?

Reply:
Yep good catch